### PR TITLE
ST for com1DFAPy

### PR DESCRIPTION
**Pull request by matthiasto**
adding Standard Tests for com1DFApy, switched to the smoother normal computation method

### CODE REVIEW

File: avaframe/com1DFAPy/com1DFA.py [hunk 1]
```diff
@@ -397,7 +397,7 @@ def initializeSimulation(cfg, demOri, releaseLine, entLine, resLine, logName, re
 
     # -----------------------
     # Initialize mesh
-    dem = initializeMesh(dem)
+    dem = initializeMesh(dem, num=8)
```
Comment (fso42) [inline avaframe/com1DFAPy/com1DFA.py:400]: Is the num option a development feature? If so, define a defaultNum at the top of the file and mark it. 
If we want this permanently, move it to the ini file

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/com1DFAPy/com1DFA.py (hunk 1 above)
Comment (matthiasto) [reply avaframe/com1DFAPy/com1DFA.py:400]: num = 4 is the default
I will add it to the ini file but then we start having a lot in the ini

**Review by fso42 — APPROVED** [review]

File: benchmarks/avaHelixChannelentTest/avaHelixChannel_com1DFACfg.ini [hunk 2]
```diff
@@ -11,10 +11,6 @@ RunCint = runBasic
 inputf = shp
 # True if full output shall be printed to terminal
 flagOut = False
-# True if entrainment areas shall be included
-flagEnt = True
```
Comment (awirb) [inline benchmarks/avaHelixChannelentTest/avaHelixChannel_com1DFACfg.ini:15]: where are the flagNoEnt=False?

Comment (awirb) [reply benchmarks/avaHelixChannelentTest/avaHelixChannel_com1DFACfg.ini:15]: HelixChannel has entrainment

File: benchmarks/avaHockeySmoothChannelentTest/avaHockeySmoothChannel_com1DFACfg.ini [hunk 3]
```diff
@@ -13,10 +13,6 @@ RunCint = runBasic
 inputf = shp
 # True if full output shall be printed to terminal
 flagOut = False
-# True if entrainment areas shall be included
-flagEnt = True
```
Comment (awirb) [inline benchmarks/avaHockeySmoothChannelentTest/avaHockeySmoothChannel_com1DFACfg.ini:17]: same here where are the flags?

File: benchmarks/avaHockeyresTest/avaHockey_com1DFACfg.ini [hunk 4]
```diff
@@ -11,10 +11,6 @@ RunCint = runBasic
 inputf = shp
 # True if full output shall be printed to terminal
 flagOut = False
-# True if entrainment areas shall be included
-flagEnt = True
```
Comment (awirb) [inline benchmarks/avaHockeyresTest/avaHockey_com1DFACfg.ini:15]: same here

File: benchmarks/avaInclinedPlaneentresTest/avaInclinedPlane_com1DFACfg.ini [hunk 5]
```diff
@@ -11,10 +11,6 @@ RunCint = runBasic
 inputf = shp
 # True if full output shall be printed to terminal
 flagOut = False
-# True if entrainment areas shall be included
-flagEnt = True
```
Comment (awirb) [inline benchmarks/avaInclinedPlaneentresTest/avaInclinedPlane_com1DFACfg.ini:15]: same here

**Review by awirb — CHANGES_REQUESTED** [review]
please add for all the ini files the flagNoEnt and flagNoRes

File: benchmarks/avaHelixChannelentTest/avaHelixChannel_com1DFACfg.ini (hunk 2 above)
Comment (matthiasto) [reply benchmarks/avaHelixChannelentTest/avaHelixChannel_com1DFACfg.ini:15]: yes so since you want the entrainment run you do not need the flagNoEnt no?

**Comment by matthiasto** [comment]
too many conflicts... Made a New branch

**Pull request closed without merging**